src/pickley/bstrap.py

```diff
@@ -160,8 +159,15 @@ def find_base(base):
     abort(f"Make sure '{candidates[0]}' is writeable.")
 
 
-def get_latest_version(package_name):
-    data = http_get(f"https://pypi.org/pypi/{package_name}/json")
+def get_latest_pickley_version(mirror):
+    if not mirror:
+        mirror = "https://pypi.org/simple"
+
+    # This is a temporary measure, eventually we'll use `uv describe` for this
+    url = os.path.dirname(mirror.rstrip("/"))
+    url = f"{url}/pypi/pickley/json"
```

Comment:
Since you use `{PICKLEY}` in the install below, how about `{PICKLEY}` here in the path component